src/main/java/org/jenkinsci/plugins/jvctb/config/CredentialsHelper.java

```diff
@@ -57,13 +57,9 @@ public static ListBoxModel doFillCredentialsIdItems(
     return fromNullable(
         CredentialsMatchers.firstOrNull(
             CredentialsProvider.lookupCredentials(
-                clazz,
```

Comment:
Add the phantom comments if you want a forced newline. Like 
```
...
clazz, //
job, //
...
```

Reply:
now that just feels like disrespecting the formatter 😢